cypress/integration/chapter/comment.spec.js

```diff
@@ -38,7 +38,7 @@ describe('Chapter comments when authenticated', () => {
 
 
         cy.get('#chapter .media-body')
-            .wait(100)
+            .wait(1000) //TODO can we do better than this wait ?🤔
```

Comment:
What do you suggest in place of this

Reply:
In the previous PR you removed it, so I dont think it has to be included here

Reply:
I am still researching on Ember behavior because the scenario is unpredictable at the moment